### PR TITLE
contributing: update contributing guide & add contributors overtime

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

+ Add contributor over time graph in README.md
+ Update outdated information in CONTRIBUTING.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,19 +53,19 @@ TiDB 中文文档的日常更新特别活跃，相应地，[TiDB 英文文档](h
```
Comment (ran-huang) [inline CONTRIBUTING.md:53]: L52: assign 自己的话，可以只回复 `/assign` 不需要加 ID

**Review by ran-huang — APPROVED** [review]
LGTM

**Review by CharLotteiu — APPROVED** [review]
LGTM

**Pull request merged**